plugins/database/src/general_query.cpp

```diff
@@ -2213,8 +2216,23 @@ checkCondInputAccess( genQueryInp_t genQueryInp, int statementNum,
                      genQueryInp.condInput.value[accessIx],
                      /*                  sessionTicket, accessControlHost, icss); */
                      sessionTicket, sessionClientAddr, icss );
-        prevStatus = status;
-        return status;
+
+        if (!std::string_view{sessionTicket, sizeof(sessionTicket) - 1}.empty()) {
```

Comment:
This is incorrect.

The constructor causes the call to `.empty()` to always return false even if the range is full of null bytes.

Reply:
Done